vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -53,7 +53,7 @@ def connect_identity
   def event_monitor_options
     @event_monitor_options ||= begin
       opts = {:hostname => self.hostname}
-      opts[:port] = MiqEventCatcherOpenstack.worker_settings[:amqp_port]
+      opts[:port] = event_monitor_class.worker_settings[:amqp_port]
```

Comment:
ok, so I think now the CI fails because of this, I'll move this to the another PR